apcd-cms/src/apps/registrations/templates/submission_form/submission_success.html

```diff
@@ -7,6 +7,8 @@ <h1>Request to Submit</h1>
     <hr />
     <p>
         Your submission was successful.
+        <br><br>
```

Comment:
No need to make changes here, but usually creating a paragraph break is done with a `<p></p> `and a css margin setting. Our sites need to be compatible with a screen reader, the `<br><br>` can trip those up and make long silences.